docs/node/guides/configuration.md

```diff
@@ -302,6 +304,33 @@ The following command will generate a new library: `nx g @nrwl/node:lib mylib`.
 
 When the `version` of `workspace.json` is set to 2, `targets`, `generators` and `executor` properties are used instead of the version 1 properties `architect`, `schematics` and `builder`.
 
+## project.json
+
+Project configurations can also be independent files, referenced by `angular.json`. For instance, an `angular.json` may contain projects configured as below.
```

Comment:
should be workspace.json not angular.json

Reply:
Thanks for catching these, I missed them when copying the statements around between the 3 versions.